lib/search/registries.rb

```diff
@@ -51,6 +51,10 @@ def organisations
       )
     end
 
+    def world_locations
+      @world_locations ||= WorldLocationsRegistry.new
```

Comment:
This is the 'entry point', in that we are switching from using a collection of rummager documents of the type 'world_location' to using an external source for world locations.

This is necessary because almost all world locations were removed from rummager and moved into the taxonomy.